docs/docusaurus/docs/core/configure_project_settings/access_secrets_managers/_aws_secrets_manager.md

```diff
@@ -0,0 +1,67 @@
+import GxData from '../../_core_components/_data.jsx'
+import PreReqFileDataContext from '../../_core_components/prerequisites/_file_data_context.md'
+
+### Prerequisites
+
+- An AWS Secrets Manager instance.  See [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/tutorials_basic.html).
+- The ability to install Python packages with `pip`.
+- <PreReqFileDataContext/>.
+
+### Procedure
+
+1. Set up AWS Secrets Manager support.
+   
+   To use the AWS Secrets Manager with {GxData.product_name} you will first need to install the `great_expectations` Python package with the `aws_secrets` requirement.  To do this, run the following command:
+
+   ```bash title="Terminal"
+   pip install 'great_expectations[aws_secrets]'
+   ```
+
+2. Reference AWS Secrets Manager variables in `config_variables.yml`.
+
+   By default, `config_variables.yml` is located at: 'gx/uncomitted/config_variables.yml' in your File Data Context.
+
+   Values in `config_variables.yml` that start with `secret|arn:aws:secretsmanager` will be substituted with corresponding values from the AWS Secrets Manager.  However, if the keywords following `secret|arn:aws:secretsmanager` do not correspond to keywords in AWS Secrets Manager no substitution will occur.
+
+   You can reference other stored credentials within the keywords by wrapping their corresponding variable in `${` and `}`.  When multiple references are present in a value, the secrets manager substitution takes place after all other substitutions have occurred.
+
+   An entire connection string can be referenced from the secrets manager:
+
+   ```yaml title="config_variables.yml"
+    my_aws_creds:  secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|connection_string
+   ```
+
+   Or each component of the connection string can be referenced separately:
+   
+   ```yaml title="config_variables.yml"
+    drivername: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|drivername
```

Comment:
For this example I think it would be much clearer if you explicitly said the secret name and the value. My first thought when reading this example was it was the secret was called `dev_db_credentials` and the values was a json object with keys `host`, `port`, `username`, etc. Except then I was confused what `connection_string` refers to. Is it another field? The name of the secret? Something else?